charts/caraml-authz/values.yaml

```diff
@@ -44,6 +44,9 @@ caramlAuthzExternalPostgresql:
 
 caraml-authz-postgresql:
   enabled: true
+  auth:
+    username: oryketo
+    database: oryketo
```

Comment:
how do we use these values? For example, we don't have it  in the mlp chart

Reply:
I had to add these because of this chart's updated Postgres db version. These will be used only when authz-specific Postgres is installed. let me also add a note in the comments for these values.

Reply:
I see. what if we upgrade PG chart to the same version we use in the other charts?

Reply:
then these new keys would come up across, I can take it up in another PR to make db versions similar across. but that would mean ensuring all the charts will successfully run after upgrades.